File: core/src/main/java/com/ustadmobile/core/view/ReportTableListComponentView.java
package com.ustadmobile.core.view;


import java.util.List;

/**
 * For presenter's access to Custom views (that implement this)
 */
public interface ReportTableListComponentView extends UstadView {


    // This defines the view name that is an argument value in the go() in impl.
    String VIEW_NAME = "ReportTableListComponentView";

    //Any argument keys:

    void setSalesLogData(List<Object> dataSet);

    void setTopLEsData(List<Object> dataSet);
}

File: core/src/commonMain/kotlin/com/ustadmobile/core/view/ReportTableListComponentView.kt
package com.ustadmobile.core.view

/**
 * For presenter's access to Custom views (that implement this)
 */
interface ReportTableListComponentView : CommonReportView {

    //Any argument keys:

    fun setSalesLogData(dataSet: List<Any>)

    fun setTopLEsData(dataSet: List<Any>)

    companion object {


        // This defines the view name that is an argument value in the go() in impl.
        val VIEW_NAME = "ReportTableListComponentView"
    }
}

